saleor/order/emails.py

```diff
@@ -65,6 +67,23 @@ def send_fulfillment_confirmation(order_pk, fulfillment_pk):
     send_templated_mail(**email_data)
 
 
+def send_fulfillment_confirmation_to_customer(order, fulfillment, user):
+    send_fulfillment_confirmation.delay(order.pk, fulfillment.pk)
+
+    events = [OrderEvent.email_sent_event(
+        order=order, source=user,
+        email_type=OrderEventsEmails.FULFILLMENT)]
+
+    # If digital lines were sent in the fulfillment email,
+    # trigger the event
+    if any((line for line in order if line.variant.is_digital())):
+        events.append(OrderEvent.email_sent_event(
+            order=order, source=user,
+            email_type=OrderEventsEmails.DIGITAL_LINKS))
+
+    return events
```

Comment:
I'm not sure about returning events here, none of the email function does that. If you wanted to encapsulate and reuse creating these two events I'd add some simple helper function only for that.